pages/apply/start.tsx

```diff
@@ -7,18 +7,41 @@ import { HeadingOne } from '../../components/content/headings';
 import ErrorSummary from '../../components/errors/error-summary';
 import Form from '../../components/form/form';
 import Layout from '../../components/layout/resident-layout';
-import { useAppDispatch } from '../../lib/store/hooks';
+import { useAppDispatch, useAppSelector } from '../../lib/store/hooks';
 import { updateBeforeFirstSave } from '../../lib/store/mainApplicant';
 import { Errors } from '../../lib/types/errors';
 import { FormID, getFormData } from '../../lib/utils/form-data';
 import processPhonenumber from '../../lib/utils/processPhonenumber';
 import { scrollToError } from '../../lib/utils/scroll';
+import {
+  ApiCallStatusCode,
+  selectPatchApplicationStatus,
+} from 'lib/store/apiCallsStatus';
+import Loading from 'components/loading';
 
 const ApplicationStartPage = (): JSX.Element => {
   const router = useRouter();
   const dispatch = useAppDispatch();
 
   const [userError, setUserError] = useState<string | null>(null);
+  const patchApplicationStatus = useAppSelector(selectPatchApplicationStatus);
+  const [hasSignedUp, setHasSignedUp] = useState<boolean>(false);
+
+  useEffect(() => {
+    if (
+      patchApplicationStatus?.callStatus === ApiCallStatusCode.FULFILLED &&
+      hasSignedUp
+    ) {
+      router.push('/apply/agree-terms');
+    }
+
+    if (patchApplicationStatus?.callStatus == ApiCallStatusCode.REJECTED) {
```

Comment:
Should this be strict ===

Reply:
yes 👍 , updated in https://github.com/LBHackney-IT/lbh-housing-register/pull/421/commits/0f9db61943508002de2b659ea834336a9978955c